test/extended/openstack/servergroup.go

```diff
@@ -6,80 +6,114 @@ import (
 
 	"github.com/gophercloud/gophercloud"
 	"github.com/gophercloud/gophercloud/openstack/compute/v2/extensions/servergroups"
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
 	g "github.com/onsi/ginkgo"
 	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	yaml "gopkg.in/yaml.v2"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
+	clientcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
 )
 
 var _ = g.Describe("[sig-installer][Feature:openstack] The OpenStack platform", func() {
 	defer g.GinkgoRecover()
 
-	// OCP 4.5: https://issues.redhat.com/browse/OSASINFRA-1300
-	g.It("creates Control plane nodes in a server group", func() {
-		ctx := context.TODO()
+	var ctx context.Context
+	var computeClient *gophercloud.ServiceClient
+	var dc dynamic.Interface
+	var oc *exutil.CLI
+	var masterInstanceUUIDs []interface{}
```

Comment:
Please move these declarations to the individual "before" and "it" blocks; it's easier to read when you only declare here the variables you want to share between steps: in this example, `masterInstanceUUIDs` and `serverGroupName`

Reply:
But then I won't be able to use it in the other test

Reply:
It's fine to redeclare these in each test:

 var ctx context.Context
	var computeClient *gophercloud.ServiceClient
	var dc dynamic.Interface
	var oc *exutil.CLI

Reply:
Do you mean to declare them again ? They are used also in BeforeEach

Reply:
Oh. Right, it makes sense to keep here what’s used in the beforeEach 👍